x-pack/plugins/security_solution/public/resolver/view/panels/panel_content_utilities.tsx

```diff
@@ -70,3 +76,60 @@ export const StyledTime = memo(styled('time')`
   display: inline-block;
   text-align: start;
 `);
+
+interface StyledCopyableField {
+  readonly borderColor: string;
+  readonly hoverBackground: string;
+}
+
+const StyledCopyableField = styled.div<StyledCopyableField>`
+  border: 0.1em solid;
+  border-color: ${(props) => props.borderColor};
+  border-radius: 3px;
+  padding: 4px;
+
+  &:hover {
+    background-color: ${(props) => props.hoverBackground};
+    color: #fff;
+  }
+`;
+
+export const CopyablePanelField = memo(
+  ({ textToCopy, content }: { textToCopy: string; content: JSX.Element | string }) => {
+    const { linkColor, pillStroke } = useColors();
+    const [isOpen, setIsOpen] = useState(false);
+    const onMouseEnter = () => setIsOpen(true);
+
+    const buttonContent = (
+      <StyledCopyableField
+        borderColor={pillStroke}
+        hoverBackground={linkColor}
+        onMouseEnter={onMouseEnter}
+      >
+        {content}
+      </StyledCopyableField>
+    );
+
+    const onMouseLeave = () => setIsOpen(false);
+
+    return (
+      <div onMouseLeave={onMouseLeave}>
+        <EuiPopover
```

Comment:
Can this be opened with the keyboard?

Reply:
Need to test it. But we may be changing the way it works...talking to ui/ux at the moment